src/foam/u2/view/ReadReferenceView.js

```diff
@@ -75,5 +78,13 @@ foam.CLASS({
       if ( dao )
         dao.find(this.data).then((o) => this.obj = o);
     }
+  ],
+  listeners: [
+    function permissionEnableLinkCheck() {
+      let permission = `${this.prop.of.id}.${this.prop.name}.disableRefLink`;
```

Comment:
You're using a permission to "disable"?

Reply:
Default for this view is to show a link - there were too many places to consider if changing this behavior. So ya, went with a permission to disable the link. Good ticket for a new higher to reverse this ...